style.css

```diff
@@ -75,15 +143,49 @@ h1 {
     padding: 5rem 2rem;
 }
 
+.portfolioHighlight {
```

Comment:
put 1rem on left and right instead of just 2rem on left so that the images stay in the center

Reply:
done =]